systemservices/ethwallet/ethwallet/list.go

```diff
@@ -0,0 +1,20 @@
+package ethwallet
+
+import (
+	"github.com/mesg-foundation/core/client/service"
+)
+
+type listOutputSuccess struct {
+	Addresses []string `json:"addresses"`
```

Comment:
replace string with `accounts.Account`?

Or at least `common.Address` as this is standard thype for address that has already implemented json encoding and more